materialize-snowflake/.snapshots/TestSQLGeneration

```diff
@@ -1,25 +1,25 @@
 --- Begin "a-schema".target_table createTargetTable ---
-  CREATE TABLE IF NOT EXISTS "a-schema".target_table (
```

Comment:
I cleaned up some extra spacing that was in these query templates, which where a lot of the diff on the file is coming from.